src/metabase/api/app.clj

```diff
@@ -8,6 +8,7 @@
     [metabase.api.collection :as api.collection]
     [metabase.api.common :as api]
     [metabase.models :refer [App Collection Dashboard Table]]
+    [metabase.models.app.graph :as graph]
```

Comment:
This conflicts with the existing `collection.graph :as graph`. Change to `:as app.graph` wherever this is required.

Reply:
In Emacs, I'm getting a warning if I use `app.graph`, but I'll tests if CI accepts it.